### PR TITLE
[1.18.2] Minimum ComputerCraft Compatibility Code

### PR DESCRIPTION
**Pull request by TechTastic**
This PR is for the minimum amount of changes necessary to fix all major incompatibilities and potential issues between VS2 and ComputerCraft (both Restitched and Tweaked)

Both have the same Mixins outside of MixinSpeakerPosition which exists exclusively on Forge

_BIg thanks to @copperwarrior for MixinSpeakerPeripheral_

- **MixinSpeakerPosition**
  - Fixes Speakers on Ships not playing sounds in worldspace but in Shipyard

- **MixinTurtleBrain**
  - Fails Movement if the turtle is in worldspace and the target position is a ship (returns "ship")
  - Fails Movement if the turtle is on ship and the target position is outsidea ship's ActiveChunkSet (returns "out of ship")

- **MixinTurtleMoveCommand**
  - Allows a Turtle to Leave Ships if the target position is outside of ship's VoxelAABB and ship is not scaled up or down

- **MixinWirelessNetwork**
  - Fixes Modems not being able to communicate from/to ships despite being within range

### CODE REVIEW

**Comment by TechTastic** [comment]
This is finally ready to be merged

It has all been tested both Forge and Fabric, in and out of dev

File: fabric/build.gradle [hunk 1]
```diff
@@ -24,6 +24,16 @@ loom {
     accessWidenerPath = project(":common").loom.accessWidenerPath
 }
 
+repositories {
+    // CC Restitched
+    maven {
+        url "https://cursemaven.com"
```
Comment (StewStrong) [inline fabric/build.gradle:30]: I think Ruby wants us to put artifacts in the VS maven. cc @Rubydesic

File: forge/build.gradle [hunk 2]
```diff
@@ -50,6 +50,16 @@ configurations {
     shadowCommon.extendsFrom core
 }
 
+repositories {
+    // CC Tweaked
+    maven {
+        url "https://cursemaven.com"
```
Comment (StewStrong) [inline forge/build.gradle:56]: Same issue as fabric

**Review by StewStrong — COMMENTED** [review]
All lgtm, though Ruby might want to change the gradle to use the vs2 maven only

**Comment by StewStrong** [comment]
We can fix the maven later, Ill just merge it

**Pull request merged**